src/toil/provisioners/clusterScaler.py

```diff
@@ -291,7 +303,13 @@ def tryRun(self):
             with throttle(self.scaler.config.scaleInterval):
                 # Calculate the approx. number nodes needed
                 # TODO: Correct for jobs already running which can be considered fractions of a job
-                queueSize = self.scaler.jobBatcher.getNumberOfJobsIssued()
+                queueSize = self.scaler.jobBatcher.getNumberOfJobsIssued(preemptable=self.preemptable)
+
+                # if we're in the non-preemptable queue, we need to see if we have any slack
+                # coming over from the preemptable queue
+                if not self.preemptable:
+                    queueSize += _preemptableQueueSlack
```

Comment:
I think we should add slack to estimatedNodes further down. The unit of queueSize is # of jobs, the unit of estimatedNodes is # of nodes.